tests/schema/tests.py

```diff
@@ -826,6 +849,120 @@ def test_alter_fk_to_o2o(self):
                     author_is_fk = True
         self.assertTrue(author_is_fk, "No FK constraint for author_id found")
 
+    def test_alter_field_fk_to_o2o(self):
```

Comment:
I think so, yes.

Databases that do no support foreign keys are still used by applications with `ForeignKey` fields. When those fields are modified to `OneToOneField`, there is still an expected migration behavior. Specifically that the index is modified to a unique constraint. This test (and the others) describe and test this behavior.